Lib/unittest/mock.py

```diff
@@ -2567,8 +2584,8 @@ def call_list(self):
 call = _Call(from_kall=False)
 
 
-def create_autospec(spec, spec_set=False, instance=False, _parent=None,
-                    _name=None, **kwargs):
+def create_autospec(spec, spec_set=False, instance=False, unsafe=False,
```

Comment:
as unsafe is being added to the API here, perhaps make it keyword only?

Reply:
Done.